app/presenters/form_list_presenter.rb

```diff
@@ -43,7 +43,7 @@ def head
   end
 
   def rows
-    forms.map do |form|
+    forms.sort_by(&:name).map do |form|
```

Comment:
What happens if there is more than one form with the same name?

Reply:
I've changed this so that now if two forms have the same name, we then sort on `created_at`